auction-deploy/requirements.txt

```diff
@@ -1,8 +1,7 @@
 click
 web3
 pendulum
-git+git://github.com/trustlines-protocol/contract-deploy-tools.git
-
```

Comment:
I think they were separated on purpose based on runtime dependencies and develop dependencies, so please give us back the blank line.